### PR TITLE
sentry logging updates

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

Some updates to our Sentry logging:

* Adds a new `KOLIBRI_DEBUG_SENTRY_ENVIRONMENT` env var to control Sentry's 'Environment'. @aronasorman you can set this to e.g. `demo-server`, `beta-server`, and `i18n-server` on our live servers. Then let's delete the `kolibridemo-*end` projects in Sentry and send all logs to `kolibri-*end`
* New info logged from frontend
  * session info
  * host
  * locale
* New info logged from backend
  * `RUN_MODE`
  * installer type
* slight improvement of dev console logging when Sentry is enabled
* upgrades sentry clients

### Reviewer guidance

* Please check this out and make sure everything looks good
* @jredrejo would you mind checking how this affects your debian changes?  I see a reference to [AIOHTTP and python 3.6](https://github.com/getsentry/sentry-python/releases/tag/0.7.7)
* @jonboiser would you mind checking the changes I made to the session action?

### References

…

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

**Review by aronasorman — COMMENTED** [review]
I can only attest to the python side, but those all look good to me!

**Comment by indirectlylit** [comment]
this is building now. Ready for review & merge

File: kolibri/core/assets/src/state/modules/core/actions.js [hunk 1]
```diff
@@ -199,9 +207,29 @@ export function blockDoubleClicks(store) {
 
 export function setSession(store, { session, clientNow }) {
   const serverTime = session.server_time;
-  setServerTime(serverTime, clientNow);
+  if (clientNow) {
```
Comment (rtibbles) [inline kolibri/core/assets/src/state/modules/core/actions.js:210]: When is this not defined, out of interest?

Comment (rtibbles) [reply kolibri/core/assets/src/state/modules/core/actions.js:210]: Oh, I see the change below, no worries!

**Review by rtibbles — APPROVED** [review]
All makes sense to me, and if the Python looks good, this is go!

**Pull request merged**